fml/platform/posix/paths_posix.cc

```diff
@@ -14,6 +14,9 @@ namespace paths {
 
 namespace {
 
+const char kFileURLPrefix[] = "file://";
```

Comment:
Here and below: not that it matters in the context, but I imagine this could be `constexpr`.

Reply:
done